cypress/integration/1-getting-started/Local tests/topbar/one/Topbar_One_1_Question_mark_button_functionalities.spec.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+// Before running this test first run the Server test 2 in order to create a user so you can log in.
+it('should be able to visit the login page', () => {
+    cy.visit(baseURL + '/') // Localhost in the env file should be changed to reflect the ONE host
+})
+
+it('should be able to log in successfully', () => {
+    cy.get('[data-test-id="input-email"]').type("cypress@test.com") // Here write your email address for your account
+    cy.get('[data-test-id="input-password"]').type("cypress123") // Write the password for your account
+    cy.get('[data-test-id="button-login-and-remember"]').click() // Here we click on the log in and remember me button
+    cy.visit(baseURL + '/') // Visiting main page of ONE
+    cy.get('.modal-header>:last-child()').click() // Here we close the start tour pop up
+})
+
+it('should be able to start the webpage tour and go through it', () => {
+    cy.get('[data-test-id="dropdown-helper"]').click() // We click on the question mark button
+    cy.get('[data-test-id="dropdown-helper-tour"]').click() // We click on the start tour button
```

Comment:
Since we're gonna be testing if the links work, just click on the skip tour button. No need to go through the whole tour.